proto/shared_constants.proto

```diff
@@ -145,11 +145,20 @@ message TestingState {
     // unknown
     unknown = 0;
 
+    // authenticated
```

Comment:
I'm second-guessing our use of the word "Testing" in TestingState... since it might not be for testing.  Would the name of the message be better served as PortState?

Reply:
So I think there are two types of "states": one is the the state in an port event, e.g., "cleared", "passed", etc; and the other is the "real" state of the port, e.g., "authenticated", "sequestered", "operational".

Here the TestingState refers to the first one. I think PortState is more suitable for the second one.

So for the first one, would it be better to name it as PortBehavior which is similar to the concept of the DeviceBehavior we already have?